app/models/course_wiki.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class CourseWiki < ApplicationRecord
```

Comment:
The naming convention in other join tables in the app is to pluralize both models: `CoursesWikis`.